scripts/generate-configs

```diff
@@ -12,8 +12,22 @@ set -euo pipefail
 
 cd "$(dirname "${BASH_SOURCE[0]}")/.."
 
-bazel build @botocore//... //gen:gen-configs
+# Output a message to stderr.
+say() {
+	echo >&2 "$@"
+}
 
-exec ./bazel-bin/gen/gen-configs "$(bazel info output_base)/external/botocore" \
-  --configs ./configs \
-  --word-frequencies ./configs/word-frequencies
+botocore="vendor/botocore/botocore/data"
+configs="./configs"
+frequencies="./configs/word-frequencies"
+
+say "Using botocore $botocore"
+say "Using configs $configs"
+say "Using word frequencies $frequencies"
+
+# Create an output link for the botocore sources.
+nix build --out-link "$botocore" '.#botocore'
```

Comment:
If `gen` works against a read-only store path, you could use `nix build --json --no-link '.#botocore' | jq -r '.[0].outputs.out'`.

Reply:
Moving the script into `writeShellScript` + `outputs.apps.*` and then just interpolating `botocore.outPath` might be preferable to calling out to `nix` multiple times in the script at any rate. Otherwise, having the out-link exist is mildly convenient if you're developing/running the generator by hand through `cabal repl` rather than having to go through `nix build/run` every time.